webapi/src/main/scala/org/knora/webapi/KnoraExceptionHandler.scala

```diff
@@ -20,12 +20,47 @@ object KnoraExceptionHandler {
 
         /* TODO: Find out which response format should be generated, by looking at what the client is requesting / accepting (issue #292) */
 
-        case rre: RequestRejectedException => complete(exceptionToJsonHttpResponse(rre, settingsImpl))
+        case rre: RequestRejectedException =>
+            extractRequest { request =>
+                val url = request.uri.path.toString
+
+                // println(s"KnoraExceptionHandler - case: rre - url: $url")
+
+                if (url.contains("v1")) {
```

Comment:
What if the URL is `/v2/resources/http://foo.bar/a3dbfv1dedc`? I think you could use `url.startsWith("/v1")` to make it safer.

Reply:
Also, should the admin routes have their own exception handler?

Reply:
> should the admin routes have their own exception handler

Probably not. At least, as long both are only returning normal JSON.